mm/src/overlays/actors/ovl_Obj_Snowball/z_obj_snowball.c

```diff
@@ -439,6 +442,7 @@ void func_80B03FF8(ObjSnowball* this, PlayState* play) {
     if (sp18->unk_04 != NULL) {
         sp18->unk_04(this, play);
     }
+    GameInteractor_Should(VB_SNOWBALL_SPAWN_ITEM, true, this, sp18->unk_00, sp18->unk_04);
```

Comment:
I think maybe this would be better below, wrapping the `if (rotY == 5)` condition, with the VB being `VB_SNOWBALL_SET_FLAG`